src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -960,6 +963,7 @@ protected override void HandleAttachRequestAsync(IRequestResponder<AttachArgumen
                 }
 
                 var properties = new Dictionary<string, object>(StringComparer.Ordinal);
+                properties.Add(DebuggerTelemetry.TelemetryMIMode, string.Empty);
```

Comment:
>string.Empty [](start = 66, length = 12)

Don't you want `miMode` here rather than `string.Empty`?

Reply:
missed a placeholder! thanks for catching it.